### PR TITLE
chore(*) report number of zones

### PR DESCRIPTION
**Pull request by nickolaev**
### Summary

Report the number of zones.

### Documentation

- Internal

### CODE REVIEW

**Comment by jakubdyszkiewicz** [comment]
Can you double check it works on Kubernetes with standalone? I remember that at some point we had a problem with retrieving list of Zones on this setup, I don't know if this is true anymore.

Change log level from V2 to V0 and see if there are any errors

**Comment by nickolaev** [comment]
> Can you double check it works on Kubernetes with standalone? I remember that at some point we had a problem with retrieving list of Zones on this setup, I don't know if this is true anymore.
> 
> Change log level from V2 to V0 and see if there are any errors

Done, it all seems OK.

**Comment by subnetmarco** [comment]
In Kubernetes will it report a number of zones of 1 or 0?

**Review by jakubdyszkiewicz — APPROVED** [review]

File: pkg/core/runtime/reports/reports.go [hunk 1]
```diff
@@ -98,6 +109,15 @@ func (b *reportsBuffer) updateEntitiesReport(rt core_runtime.Runtime) error {
 		return err
 	}
 	b.mutable["meshes_total"] = strconv.Itoa(len(meshes.Items))
+
+	zones, err := fetchZones(rt)
+	if err != nil {
+		return err
+	}
+	if len(zones.Items) > 0 {
```
Comment (nickolaev) [inline pkg/core/runtime/reports/reports.go:117]: @subnetmarco if there are no zones, it will not send the `zones_total` field.

**Pull request merged**